apps/web/components/EnhancedActiveLink.tsx

```diff
@@ -15,7 +15,7 @@ export const EnhancedActiveLink = React.forwardRef(function ActiveLink(
 ) {
   const router = useRouter()
   const active = router.asPath.startsWith(props.as ?? props.href)
-  const exactActive = (props.as ?? props.href) === router.asPath
```

Comment:
what was this change about?

Reply:
`router.asPath` includes query params so it wasn't matching the link and highlighting the nav links like intended. Using `.pathname` strips the query params